load_balancer.tf

```diff
@@ -100,7 +100,8 @@ resource "aws_acm_certificate" "production" {
 }
 
 resource "aws_route53_record" "cert_validation" {
-  for_each = {
+
+  for_each = length(var.lb_domain_name) == 0 ? {} : {
```

Comment:
logic in a tf file 😱

Reply:
yeah its all over the place i wish it was easier to write conditionals